tasks/install.yml

```diff
@@ -80,6 +85,7 @@
   when:
     - ansible_os_family | lower == "debian"
     - restic_user != 'root'
+    - restic_group != 'root'
```

Comment:
@TheLastProject I think this line is wrong too, right? `libcap2-bin` should be installed if restic_user OR restic_group are not 'root', like this:
```
 - restic_user != 'root' or restic_group != 'root'
```

Reply:
I think this is @paulfantom's code, but it seems to me that that indeed should be changed.